runner/src/gdb.rs

```diff
@@ -26,7 +26,7 @@ static GDB_EXECUTABLES: &[&'static str] = &[
 /// need for such a function.
 fn build_gdb_command(gdb_executable: &str) -> Command {
     // Retrieve the path of Miralis's binary
-    let mut miralis_path = get_target_dir_path(&Target::Miralis);
+    let mut miralis_path = get_target_dir_path(&Target::Miralis, Mode::Debug); // Could there be any cases when we want to invoke gdb on a release mode build?
```

Comment:
Yes we might want to run GDB with a release builds, for instance to debug an issue that appears only when compiling in release mode (which can happen in projects with `unsafe` rust code when we do not properly enforce the safety requirements).